reference/auth/models/consent/records.v1.yaml

```diff
@@ -0,0 +1,21 @@
+title: records.v1
+x-stoplight:
+  id: 0tuzybru8eee5
+type: object
+properties:
+  data:
+    type: array
+    x-stoplight:
+      id: whbezmg3q7pxh
+    uniqueItems: true
+    items:
+      $ref: ./record.v1.yaml
+      x-stoplight:
+        id: u1b4u8fpl1ed6
+  count:
```

Comment:
Is this useful? Isn't it just a data.length? Do we really need to put this in the response when it's so easy for a client to calculate it itself?

PS- I believe a count appears in the list of consent types too.

Reply:
This is not the length of the data array. Count is the total number of documents matching the query.